tests/rbd/test_rbd_group_commands_validation.py

```diff
@@ -0,0 +1,329 @@
+                else:
+                    log.error(f"{pool}/{image} listed in the {pool}/{group} image list")
+                    return 1
+
+            # Add image back to the group
+            out, err = rbd.group.image.add(
+                **{"group-spec": f"{pool}/{group}", "image-spec": f"{pool}/{image}"}
+            )
+            if err:
+                log.error(
+                    f"Adding image {pool}/{image} to group failed with error {err}"
+                )
+                return 1
+            else:
+                log.info(
+                    f"Successfully added image {pool}/{image} to group {pool}/{group}"
+                )
+
+            # Rename the group
+            group_new = "group_new" + random_string(len=4)
+            out, err = rbd.group.rename(
+                **{
+                    "source-group-spec": f"{pool}/{group}",
+                    "dest-group-spec": f"{pool}/{group_new}",
+                }
+            )
+            if err:
+                log.error(f"Rename group {pool}/{group_new} failed with error {err}")
+                return 1
+            else:
+                log.info(f"Successfully renamed group to {pool}/{group_new}")
+
+            # Create group snap
+            snap = "snap_" + random_string(len=4)
+            out, err = rbd.group.snap.create(
+                **{"group-spec": f"{pool}/{group_new}@{snap}"}
+            )
+            if err:
+                log.error(
+                    f"Create snap {pool}/{group_new}@{snap} failed with error {err}"
+                )
+                return 1
+            else:
+                log.info(f"Successfully created snap {pool}/{group_new}@{snap}")
+
+            # list group snap
+            out, err = rbd.group.snap.list(**{"group-spec": f"{pool}/{group_new}"})
+            if err:
+                log.error(f"Group snap list failed with error {err}")
+                return 1
+            else:
+                if snap in out:
+                    log.info(f"Snap {snap} listed in group snap list")
+                else:
+                    log.error(f"Snap {snap} not listed in group snap list")
+                    return 1
+
+            # Delete the snap
+            out, err = rbd.group.snap.rm(**{"group-spec": f"{pool}/{group_new}@{snap}"})
+            if err:
+                log.error(
+                    f"Delete group snap {pool}/{group_new}@{snap} failed with error {err}"
+                )
+                return 1
+            else:
+                log.info("Successfully deleted grop snap {pool}/{group_new}@{snap}")
+
+            # list group snap
+            out, err = rbd.group.snap.list(**{"group-spec": f"{pool}/{group_new}"})
+            if err:
+                log.error(f"Group snap list failed with error {err}")
+                return 1
+            else:
+                if snap not in out:
+                    log.info(f"Group snap {snap} not listed in group snap list")
```

Comment:
please add `Group snap {snap} not listed in group snap list after successful deletion`

Reply:
Addressed